src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -31,9 +31,17 @@ public LeaseClientProvider getLeaseClientProvider() {
 
     @Bean
     public BlobServiceClient getStorageClient(
-        StorageSharedKeyCredential credentials,
-        @Value("${storage.reform.connection-string}") String connectionString
+        @Value("${storage.account-name}") String accountName,
+        @Value("${storage.account-key}") String accountKey,
+        @Value("${storage.url") String storageUrl
     ) {
+        String connectionString = String.format(
+            "DefaultEndpointsProtocol=https;BlobEndpoint=%s;AccountName=%s;AccountKey=%s",
+            storageUrl,
+            accountName,
+            accountKey
+        );
+
```

Comment:
I think we do not need to create connection string ,we can create the client by directly setting values.

Reply:
I think that's what Nitin tried initially and it didn't work. Is this correct @nitinprabhu ?

Reply:
> I think we do not need to create connection string ,we can create the client by directly setting values.

what do you mean @mdayican ? example ?

Reply:
I'm gonna merge this now, as this unblock production deployment. We can change the way we build the client at any point.